ocs_ci/helpers/helpers.py

```
@@ -1312,6 +1312,169 @@ def get_cephfs_data_pool_name():
+        if pool.get("name") == pool_name:
+            patch = [
+                {
+                    "op": "remove",
+                    "path": (
+                        "/spec/managedResources/cephFilesystems"
+                        f"/additionalDataPools/{idx}"
+                    ),
+                }
+            ]
+            storage_cluster_obj.patch(params=patch, format_type="json")
+            logger.info(f"CephFS data pool '{pool_name}' deleted")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)